src/EditorFeatures/Test/Diagnostics/AbstractUserDiagnosticTest.cs

```diff
@@ -358,7 +366,7 @@ protected Document GetDocumentAndAnnotatedSpan(TestWorkspace workspace, out stri
             if (!hasProjectChange)
             {
                 // If there is just one document change then we expect the preview to be a WpfTextView
-                var content = (await editHandler.GetPreviews(workspace, operations, CancellationToken.None).GetPreviewsAsync())[0];
+                var content = (await editHandler.GetPreviews(workspace, operations, cancellationToken: CancellationToken.None).GetPreviewsAsync())[0];
```

Comment:
yeah, i'll clean up.  sorrya bout this.

(to answer your question.  Yes, there was a point these got added.  But it's not needed now).